### PR TITLE
Show Pokemon types in map popups

### PR DESCRIPTION
**Pull request by UzEE**
## Description
Added the ability to show Pokemon types in the map popups.

## Motivation and Context
It would be very helpful to have the ability to show Pokemon types in their popups, to make it easier to hunt down and track certain types of Pokemon. I've added type labels to Pokemon marker popups, and will also send in another pull request tomorrow that adds the ability to set notifications for certain types.

For example, it would be possible to get an alert notification whenever a Dragon or Fairy type Pokemon spawns in the search area.

## How Has This Been Tested?
This PR only adds a couple of lines of code to the Pokemon marker label code. All the styles are inline and I've tested the code in Chrome (Windows, iOS), Edge (Windows), and Safari (iOS).

## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by tim95030** [comment]
Can we merge this into my pull request? As I have all the type data loaded into the json files.

See #1899

File: static/map.js [hunk 1]
```diff
@@ -352,8 +369,9 @@ function pokemonLabel(name, disappear_time, id, latitude, longitude, encounter_i
             <small>
                 <a href='http://www.pokemon.com/us/pokedex/${id}' target='_blank' title='View in Pokedex'>#${id}</a>
             </small>
+            <span> - </span>
+            <small>` + typeStr + `</small>
         </div>
-        <div>
```
Comment (redprince) [inline static/map.js]: intended? seems like a typo :D

Comment (oceanor) [inline static/map.js]: intended? seems like a typo :D

Comment (UzEE) [inline static/map.js]: Good catch. Must have happened when squashing the commits.

**Comment by UzEE** [comment]
@tim95030 Ah I didn't see your PR before. I wanted to avoid storing types in the JSON files since it wasn't as efficient as packing them into a numeric array, which is smaller in size. Also because I don't know python, I wanted to avoid fiddling around with the server side code.

If everyone feels this would be a better fit in #1899 then, we can tweak the UI code to use the JSON data.

**Comment by chaze117** [comment]
Nice feature, by the way I wrote a rarity script based on your code. Thanks for sharing.

**Comment by tim95030** [comment]
I have integrated this to #1899

**Comment by UzEE** [comment]
Great. I guess this should be closed now then?

**Comment by Crazy-Drew** [comment]
This would be a great feature I think, I like it.

**Pull request closed without merging**